### PR TITLE
Romy sensor

### PR DESCRIPTION
**Pull request by xeniter**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/112388
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/romy.markdown [hunk 1]
```diff
@@ -35,6 +35,20 @@ Currently supported services are:
 - `stop`
 - `return_to_base`
 
+## Integration entities
+
+The **Romy** {% term integrations %} will add the following sensors.
```
Comment (frenck) [inline source/_integrations/romy.markdown]: - This is not the first time this term has been introduced.
- No need to emphasize the brand name here?
- 
```suggestion
The Romy will add the following sensors.
```

File: source/_integrations/romy.markdown [hunk 2]
```diff
@@ -35,6 +35,20 @@ Currently supported services are:
...
+
+Sensors:
+
+- **Battery Level**: The battery level in percent
+- **RSSI**: Wi-Fi signal strength
+- **Dustbin Dirt Level**: The raw dustbin dirt sensor value (not every ROMY has one)
+- **Total cleaning time**: How many hours the robot has spent cleaning in total
+- **Total cleaning runs**: The total number of cleaning runs
+- **Total cleaned area**: The total area in m² the robot has cleaned
+- **Total distance driven**: The total distance in m the robot has driven
```
Comment (frenck) [inline source/_integrations/romy.markdown]:
```suggestion
- **Battery Level**: The battery level in percent.
- **RSSI**: Wi-Fi signal strength.
- **Dustbin Dirt Level**: The raw dustbin dirt sensor value (not every ROMY has one).
- **Total cleaning time**: How many hours the robot has spent cleaning in total.
- **Total cleaning runs**: The total number of cleaning runs.
- **Total cleaned area**: The total area in m² the robot has cleaned.
- **Total distance driven**: The total distance in m the robot has driven.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/romy.markdown [hunk 3]
```diff
@@ -35,6 +35,20 @@ Currently supported services are:
 - `stop`
 - `return_to_base`
 
+## Integration entities
+
+The Romy will add the following sensors.
```
Comment (frenck) [inline source/_integrations/romy.markdown]:
```suggestion
The ROMY will add the following sensors.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by joostlek — DISMISSED** [review]
There's a merge conflict

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @xeniter :+1:

**Pull request merged**